DISTRO_PACKAGE.md

```diff
@@ -0,0 +1,74 @@
+# netavark: A container network stack
```

Comment:
I know that you got this from the readme.md, but we should really cap the first letter here, or not cap it down below.  I'd vote for capping, ala Podman instead of podman